### PR TITLE
reformat programme's page

### PR DESCRIPTION
**Pull request by SimonNtz**
Redo old PR without the commits.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -23,19 +23,19 @@ body {
 
 .programme_style h3 {
   text-transform: uppercase;
-  text-decoration: underline;
   text-align: center;
-  color: #000099;
+  color: #0066ff;
```
Comment (rspt) [inline main.css]: Use SCSS or CSS3 variables for that please. And name the color!

File: programme/index.html [hunk 2]
```diff
@@ -2,126 +2,130 @@
...
-</p>
-
-<h3>FINANCES</h3>
-<h4><i>Centimes additionnels</i></h4>
-<p>
-  Tout programme passe par de bonnes finances et c'est la raison pour laquelle nous entendons maintenir au niveau actuel le niveau de prélèvement des centimes addi-tionnels en Ville de Genève et dans un certain nombre de communes de l'agglomé-ration.
-  Dans cet esprit, nous nous élevons contre la suppression de la taxe profes-sionnelle annoncée par le Conseil d'Etat qui rapporte, bon an mal an, à la seule Ville de Genève, un montant de l'ordre de 100 millions, sa suppression n'aurait que peu d'effet sur les entreprises et ne vise qu'à satisfaire une clientèle électorale.
-  Nous refusons également l'imposition sur le lieu de domicile qui, si elle était appliquée, réduirait également les recettes des communes. Enfin, pour conclure, notre asso-ciation s'oppose résolument à la réforme des entreprises (RIE 3) qui réduirait nota-blement les recettes fiscales des communes et de la Ville de Genève, en particulier.
-</p>
+<div class=programme_style>
```
Comment (rspt) [inline programme/index.html]: You miss the `"`

`class="whatever_the_class"`

File: main.css (hunk 1 above)
Comment (SimonNtz) [reply main.css]: Ok. First time I use css variables. Check if the colors's name is good plz !

File: programme/index.html (hunk 2 above)
Comment (SimonNtz) [reply programme/index.html]: Crap

**Pull request merged**